cubeb-sys/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Dan Glastonbury <dglastonbury@mozilla.com>"]
 repository = "https://github.com/mozilla/cubeb-rs"
 license = "ISC"
 description = "Native bindings to the cubeb library"
-exclude = ["libcubeb/googletest/"]
+exclude = ["libcubeb/"]
```

Comment:
I think this is not correct as-is: docs.rs doesn't have a network connection, so it'll fail to clone this.